va/va_fei_h264.h

```diff
@@ -318,6 +318,115 @@ typedef struct _VAEncFEIDistortionH264 {
     uint32_t    reserved1[2];
 } VAEncFEIDistortionH264;    // 48 bytes
 
+/** \brief Motion Vector and Statistics frame level controls.
+ * VAStatsStatisticsParameterBufferType for H264 16x16 block
+ **/
+typedef struct _VAStatsStatisticsParameterH264
+{
+   VAStatsStatisticsParameter stats_params;
+
+    uint32_t    frame_qp                    : 8;
+    /** \brief maximum number of Search Units, valid range is [1, 63] */
+    uint32_t    len_sp                      : 8;
```

Comment:
How large is the search unit? what does sp stand for?

Reply:
The SP length range is [1, 63] as the comment, SP means search path